src/services/mailServer.ts

```diff
@@ -20,9 +21,12 @@ export const createPayload = (payload: IMailNode) => {
 };
 
 export const sendMail = async (payload: IMailNode) => {
+  const secretValue = await getSecretValue();
+
   return await http.post(mailConfig.mailServer.host, createPayload(payload), {
     headers: {
-      'x-api-key': mailConfig.mailServer.password
+      'x-api-key':
+        secretValue['MAIL_SERVER_PASSWORD'] || mailConfig.mailServer.password
```

Comment:
Add the value `secretValue` in config file of mail config. 

```js
{
  password: secretValue['MAIL_SERVER_PASSWORD'] || process.env.MAIL_PASSWORD || ''
}
```

Reply:
doesn't this make the whole config file async?